api/controllers/logging.go

```diff
@@ -0,0 +1,41 @@
+package controllers
+
+import (
+	"net/url"
+	"strconv"
+	"time"
+
+	"github.com/quintilesims/layer0/common/errors"
+)
+
+func parseLoggingQuery(query url.Values) (int, time.Time, time.Time, error) {
```

Comment:
Shouldn't we test this?

Reply:
yep, it's covered in the [service test](https://github.com/quintilesims/layer0/blob/service-logs/api/controllers/service_test.go#L138) and [task test](https://github.com/quintilesims/layer0/blob/service-logs/api/controllers/task_test.go#L143)